fixes #1776 (#1788)
exercism/go: exercises/concept/party-robot/.docs/instructions.md

## 3. Give directions

> Implement the `AssignTable` function to give directions. The robot provides the table number in a 3 digits format. If the number is less than 3 digits it gets extra leading zeroes to become 3 digits (eg. 3 becomes 003). The robot mentions too the distance of the table with but only with a precision that's limited to 1 digit.

Reviewer: ```suggestion
Implement the `AssignTable` function to give directions. The robot provides the table number in a 3 digits format. If the number is less than 3 digits it gets extra leading zeroes to become 3 digits (eg. 3 becomes 003). The robot also mentions the distance of the table. Fortunately only with a precision that's limited to 1 digit.
```
